index.html

```diff
@@ -0,0 +1,236 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="AppBar">
+        <header class="ToolBar">
+          <nav>
+            <img src="images/NormalButton.png" alt="" class="NormalButton" />
+          </nav>
+          <div class="Logo">
+            <a href="#">faishar faike</a>
+          </div>
+        </header>
+      </div>
+      <div class="headline">
+        <div class="content">
+          <div class="PrimaryText">
+            <h1 class="PrimaryText">Hey There. I'm faishar</h1>
+          </div>
+          <div class="SupportingText">
+            <p class="SupportingText">I'm a Software Developer</p>
+          </div>
+          <div class="discription">
+            <p class="discription">
+              I can help you build a product , feature or website Look through
+              some of my work and experience! If you like what you see and have
+              a project you need coded, don't hestiate to contact me.
+            </p>
+          </div>
+          <ul class="SocialMedia">
+            <li>
+              <a href="https://github.com/Robison-faike123">
+                <img src="images/Icon -GitHub.png" alt="-GitHub" />
+              </a>
+            </li>
+            <li>
+              <a href="https://www.linkedin.com/">
+                <img src="images/Icon - Linkedin.png" alt="Linkedin" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="images/Icon - Angelist.png" alt="Angelist" />
+              </a>
+            </li>
+            <li>
+              <a href="https://twitter.com">
+                <img src="images/Icon - Twitter.png" alt="Twitter" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="images/Icon - Medium.png" alt="Medium" />
+              </a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    <!--this is for second section -->
+    <section class="Works">
+      <div class="RecentWorks">
+        <h1>My Recent Works</h1>
```

Comment:
- [ ] For semantic reasons, it is always advised to only have one ```<h1>``` tag per webpage. Since you already have it, I would strongly advise you to change this to a hx maybe even a h2 would be good.

**I have noticed you used it multiple other places, Kindly make this change in all those places as well**

Reply:
I have worked on it and now h tags are from h1-h4 in all project